### ISSUE TITLE
Spine JSON support.

### ISSUE THREAD

**Issue by cantsin** [SixSecondMonks/sos-display#47]
If we convert the Kinect data to [Spine JSON](http://esotericsoftware.com/spine-json-format) we might be able to use built in stuff such as Spineboy for easier animation of avatars. See also [this overview](http://esotericsoftware.com/spine-in-depth)

**Comment by Iteratix** [comment]
Looked into this, not bad. The trial version of Spine doesn't allow for import so we'd probably need to purchase Spine to really play with it.

Correct me if I'm wrong but the advantage of using Spine/SpineJSON is that we can record a session with the Kinect, then import into Spine to play with? How would we roundtrip our improvements back into sos-display or the new game?

**Comment by cantsin** [comment]
Right now we have no way to create avatars automatically. All we have is the skeletal data, and even that is pretty rigid. I'm thinking primarily about following the Spine spec rather than coming up with our own ad hoc format. Thought experiment: if we added a Ganesha avatar, what would that look like? It appears to me (at least superficially) that having a predefined spec would be nice.

**Comment by cantsin** [comment]
Of course we would still need some glue code between the Kinect skeleton and the Spine format but this might be easier _if_ we end up doing more than one avatar. Something to think about.

**Comment by Iteratix** [comment]
Yeah, I'm sold really -- as the SpineJSON format seems to be supported in many different places. So, should we convert to SpineJSON on the server side...
